### PR TITLE
add script for getting the screenshot diffs between prod and autopush

### PR DESCRIPTION
**Pull request by chejennifer**
This tool gets the link to see the diffs between screenshots taken against autopush.datacommons.org between the last autopush that was pushed to prod & the current autopush

### CODE REVIEW

File: tools/screenshot_diff/diff.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+prod_commit_date=$(git log -1 --date=format:"%Y_%m_%d_%H_%M_%S" --format="%ad" prod)
```
Comment (shifucun) [inline tools/screenshot_diff/diff.sh:18]: when I try line 18, got a fatal error:

fatal: ambiguous argument 'prod': unknown revision or path not in the working tree.

**Review by shifucun — COMMENTED** [review]
this is very nice!!

File: tools/screenshot_diff/diff.sh (hunk 1 above)
Comment (chejennifer) [reply tools/screenshot_diff/diff.sh:18]: hm, I wonder if you need to pull tags first, can you try on a clean master:
- run `git pull origin --tags` (origin or dc or what you use when you do `git pull x master`)
- then run the script

Comment (shifucun) [reply tools/screenshot_diff/diff.sh:18]: okay, this works! 
I wonder how is "prod" set in Github? And can we also add "git pull" command in this script?

**Review by shifucun — APPROVED** [review]

File: tools/screenshot_diff/diff.sh (hunk 1 above)
Comment (chejennifer) [reply tools/screenshot_diff/diff.sh:18]: updating the "prod" tag is actually part of the release process: https://screenshot.googleplex.com/BDEjF2MVMnzxYQi

I think the problem with adding git pull command in the script is that some people use `origin` and some people use `dc` .. but maybe I can add that as a step in the README?

Comment (shifucun) [reply tools/screenshot_diff/diff.sh:18]: Adding to documentation sg!

For prod tag, I have been missing that (not strictly following the steps). We can maybe monitor how this goes and decide if need to use this https://gist.github.com/rponte/fdc0724dd984088606b0 later.

Comment (chejennifer) [reply tools/screenshot_diff/diff.sh:18]: updated & ok, we can monitor!

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**